# HIV-1 and human genetic variation
<https://www.nature.com/articles/s41576-021-00378-0>
* Paul J. McLaren & Jacques Fellay
* Nature Reviews Genetics (2021)
* Published: 24 June 2021

Sections that I am not reviewing, include the following notesL

Variations in probability of infection.
In this they go through the host genetics role for this probability.
Fig 1. 
Host CCR5 deletion modulating viral binding.
Cell cycle, including binding, release of RNA, reverse transcription into viral DNA, and integration into human host.
Possible to inhibit the viral parts of life cycle but not possiible to treat the integrated DNA part of life cycle.
I am sure there are many papers about crispr type treatments in the future.
Viral proteins degrated by proteasome and presented on MHC class I (encoded by HLA class I (HLA-B*57:01, HLA-C, HLA-A)).
Presented to cytotoxic lymphocytes (CTL), and thereby eliminated by host immune response. 
Many candidate gene studies done but most of them discovered not to be valid. 
Only the CCR5 and HLA genes were statistically valid for assocciation in these early studies, and first GWAS. 




## Within-host diversity and epitope presentation

Impact of specific class I HLA alleles on HIV progression
* population level
* individual level (within-host diversity)

Individuals carrying two different alleles at all three class I genes associated with a reduced time to AIDS.

Supported by a GWAS: individuals carrying different HLA alleles at each class I gene had a significantly lower viral load than homozygous individuals.

Heterozygote advantage: likely due to presenting more HIV epitopes than homozygote.

In silico study
* the binding affinity of all possible 9-mer peptides in the HIV proteome 
* to HLA proteins encoded by the different class I alleles
*  set point viral load (spVL) was negatively correlated with the breadth of the peptide repertoire bound by an individual’s HLA protein isoforms.
* However, quantity of epitopes alone is unlikely to fully explain the protective capacity of an individual’s HLA alleles.
* Subsets of epitopes that are uniquely presented by protective HLA isoforms explainedmore than the entire predicted set. 
* Furthermore, HIV epitopes that encode structurally important residues are preferentially targeted by protective HLA isoforms 
	- associate with elite control.

_Quantity and quality of HIV epitopes presented by combinations of HLA isoforms are the key drivers of spVL._

Figure 2
![Figure 2](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41576-021-00378-0/MediaObjects/41576_2021_378_Fig2_HTML.png)
> a | HIV-infected cells expressing protective HLA-B alleles tend to present a more diverse and more structurally conserved set of HIV epitopes compared to non-protective alleles. Interactions with protective alleles tend to produce a more polyfunctional cytotoxic T lymphocyte (CTL) response. b | HLA-C protein isoforms vary broadly in their level of expression on the surface of infected cells. HLA-C alleles that do not have a binding site for microRNA-148a (miRNA-148a) in the 3′ untranslated region of their mRNAs escape suppression and present more peptide on the cell surface than alleles with an miRNA-148a binding site, resulting in the initiation of stronger CTL responses. c | Different HLA-A alleles express different amounts of HLA-A signal peptide, which positively correlates with HLA-E peptide expression. HLA-E interacts with the NKG2A receptor on the surface of natural killer (NK) cells and, when highly expressed, inhibits the killing of infected cells.

## Non-classical effects of HLA variation

Variable expression levels of classical HLA-C alleles linked to HIV control.

* High expression protects against disease progression.
	- across ancestries 
	- linked to the absence of a variable microRNA-148a (miR-148a) binding site in the 3′ untranslated region of HLA-C.
	- proposed: mRNA lacking miR-148a binding site are not suppressed, load more HIV epitopes, and detected by CTL (**Figure 2b**).


HLA-A expressed at variable level on cell surface.
* High levels associate with poorer control of viral replication,
* faster disease progression.
* Proposed: enhanced production of the HLA class I signal peptide that regulates HLA-E expression
	- HLA-E is a ligand for natural killer group protein 2A (NKG2A) 
	- interaction results in strong inhibition of natural killer (NK) cell degranulation (**Figure 2c**).
* Enhanced production of the HLA class I signal peptide may lead to enhanced inhibition of immune responses. 

The combination of HLA genotype and the expression of particular killer cell immunoglobulin-like receptor (KIR) proteins variably modulated HIV disease course.
*  KIR proteins - highly variable cell-surface receptors 
	- expressed on NK cells (some T cells)
	- when engaged by their cognate receptors, either activate or inhibit NK cell-mediated killing 
* Combination of the activating KIR3DS1 allele with a set of HLA-B alleles that carry isoleucine in the Bw4 epitope (Bw4-I80) is highly associated with HIV control.

_These are examples of the complex interplay between epitope presentation, HLA protein expression and NK inhibition._

Figure 2 - repeated

![Figure 2](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41576-021-00378-0/MediaObjects/41576_2021_378_Fig2_HTML.png)
> a | HIV-infected cells expressing protective HLA-B alleles tend to present a more diverse and more structurally conserved set of HIV epitopes compared to non-protective alleles. Interactions with protective alleles tend to produce a more polyfunctional cytotoxic T lymphocyte (CTL) response. b | HLA-C protein isoforms vary broadly in their level of expression on the surface of infected cells. HLA-C alleles that do not have a binding site for microRNA-148a (miRNA-148a) in the 3′ untranslated region of their mRNAs escape suppression and present more peptide on the cell surface than alleles with an miRNA-148a binding site, resulting in the initiation of stronger CTL responses. c | Different HLA-A alleles express different amounts of HLA-A signal peptide, which positively correlates with HLA-E peptide expression. HLA-E interacts with the NKG2A receptor on the surface of natural killer (NK) cells and, when highly expressed, inhibits the killing of infected cells.

Other people discussed the next sections:
Figure 3.
Dosage of CCR5 delta32, heterozygous protection, and homozygous are resistant to infections.
Control of expression of a non-coding mRNA which protects CCT5 from degredation; higher expression, versus those with a SNP that lowers expression of the mRNA which in turn reduces CCR5 expression. 


## HIV precision medicine
### Expanding antiretroviral therapy to eradicate HIV. 
* Now accepted that non-detectable plasma viral loads cannot transmit.
* Joint United Nations Programme on HIV/AIDS (UNAIDS) aim:
	- 90% of infected know status, 
	- 90% of those on antiviral therapy,
	- 90% of those suppressing the virus below the level of detection.
	- ~34 million on life-long chometherapy.

### HIV pharmacogenetics
Figure 5 shows the different mechanisms of pharma, including:

* Entry inhibitors, 
	- prevent the binding of the viral spike protein gp120 to host cell receptors and fusion of the virus with host cell membranes
* nucleoside and non-nucleoside reverse transcriptase inhibitors
	- prevent the reverse transcription of the viral RNA genome into DNA
* integrase inhibitors
	- prevent the integration of the viral DNA product into the host genome
* protease inhibitors
	- prevent the cleavage of viral polyproteins into their functional subunits

Human genetic variability is known to influence response to some drugs.

### Figure 5
[Figure 5 with legend](https://www.nature.com/articles/s41576-021-00378-0/figures/5)

![Figure 5](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41576-021-00378-0/MediaObjects/41576_2021_378_Fig5_HTML.png)
> Commonly used antiretroviral drugs target receptor binding, reverse transcription, integration and protease cleavage. Genetic variation in several human genes (in bold) have been shown to modify drug metabolism and contribute to adverse drug reactions (Table 1). CCR5, CC-chemokine receptor 5.

### Table 1 
**Genetic variants known to affect the pharmacokinetics of anti-HIV drugs**

| Drugs | Class | Genes | Variants | Effect on patients |
|:---|:---|:---|:---|:---|
| Abacavir | NRTI | HLA-B | rs2395029 (*57:01) | Hypersensitivity reaction |
| Atazanavir | PI | UGT1A1 | rs8175347 (*28,*37) | Predisposition to hyperbilirubinemia |
| Efavirenz | NNRTI | CYP2A6 | rs1801272 (*2), rs5031016 (*7,*10,*19), rs28399433 (*9) | Predisposition to high drug plasma levels and treatment discontinuation |
| Efavirenz and Nevirapine | NNRTI | CYP2B6 | rs3745274 (*6), rs12721655 (*8,*13), rs35303484 (*11), rs36060847 (*12), rs35773040 (*14), rs35979566 (*15), rs28399499 (*16,*18) | Predisposition to high drug plasma levels |
| Etravirine | NNRTI | CYP2C9, CYP2C19 | rs1057910 (*3), rs4424285 (*2) | Predisposition to high drug plasma levels |
| Lopinavir | PI | ABCC2, CYP3A | rs717620 (T), rs6945984 (C) | Predisposition to high drug plasma levels |
| Lopinavir | PI | SLCO1B1 | rs4149056 (*5), rs17329885 (*4) | Predisposition to low drug plasma levels |

NNRTI, non-nucleoside reverse transcriptase inhibitor; NRTI, nucleoside reverse transcriptase inhibitor; PI, protease inhibitor.

Example, the allele CYP2B6*6 (rs3745274):
* Results in the slow metabolism of efavirenz and nevirapine, 
	- two non-nucleoside reverse transcriptase inhibitors recommended for first-line use by WHO until recently, 
* ~2x higher frequency in some African populations compared to Europeans.
* led to thousands of cases of treatment discontinuation in Zimbabwe when the nation adopted a single-pill efavirenz-containing regimen.
### Example from a paper we reviewed
[https://miro.medium.com/max/700/1*EWi6AcuXwRNw8tZiouycVA.gif](https://miro.medium.com/max/700/1*EWi6AcuXwRNw8tZiouycVA.gif)

["Sequencing of 53,831 diverse genomes from the NHLBI TOPMed Program" (Taliun et al., 2021)](https://lawlessgenomics.com/topic/stargazing)
Used a method of known pharmacogene (PGx) variation annotation that can be adapted for many other drug-gene database applications. 
The star allele nomenclature method is applied to large scale data to screen for possible pharmacogenomic interactions. 
Annotate allelic variation in genes impacting drug metabolism, disposition and response (e.g. of interest to clinicians prescribing medications). 
The Pharmacogene Variation (PharmVar) consortium repository is used to label human cytochrome P450 (CYP) genes for known PGx variation.
A full list of cytochrome P450 (CYP) alleles with star notation can be found via the
[Pharmacogene Variation (PharmVar)](https://www.pharmvar.org/genes) consortium.

Taliun et al. used _de novo_ genome assembly for
* previously observed haplotype variation
* SNVs
* indels
* copy number and
* homology information
to identify and classify haplotypes in interesting regions of the genome.

They illustrate this by focusing on 
one such region around the _CYP2D6_ gene.
This encodes an enzyme that metabolizes approximately 25% of prescription drugs 
and the activity of which varies substantially among individuals.
More than 150 _CYP2D6_ haplotypes have been described,
some involving a gene conversion with its nearby non-functional
but highly similar paralogue _CYP2D7_.

They performed _CYP2D6_ haplotype analysis for all 53,831 TOPMed individuals.
Called a total of 99 alleles (66 known and 33 novel) representing:
* increased function,
* decreased function and
* loss of function (Supplementary Table 12).

Nineteen known alleles and
all novel alleles
were defined by structural variants,
including complex CYP2D6-CYP2D7 hybrids and
extensive copy number variation,
which ranged from zero to eight gene copies (Supplementary Figs. 27, 28).

### Complex trait genomics in HIV medicine.
* In addition to direct interactions between host genotype and drug metabolism
* also develope chronic diseases
	- cardiovascular disease
	- metabolic syndrome
	- kidney disease
	- liver fibrosis
* all known to have high heritability in the HIV uninfected population
* some are enhanced when on therapy for HIV

* A push to develop PRS in the general population
* Only starting in HIV treatment studies
* Addition of a PRS to the known clinical and pharmacological risk factors may benefit
	- cardiometabolic diseases 
	- chronic kidney disease

* PRS are not necessarily transferable across ancestral groups
* As in all areas of genomics, attention should be paid to enhancing diversity.


URL : <https://www.nature.com/articles/s41576-021-00378-0>
